From 1479bda8d36ed7b167a003e016b426ae615c6f33 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Wed, 25 Sep 2019 10:08:25 +0100
Subject: [PATCH 1/2] configure: Use PKG_PROG_PKG_CONFIG instead of reinventing
 it

When cross-compiling (for example for aarch64 on x86), using AC_PATH_PROG
to search for pkg-config will find the build architecture (x86)
pkg-config, but what we want is the host architecture (aarch64)
pkg-config. PKG_PROG_PKG_CONFIG does this correctly, by using
AC_PATH_TOOL.

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 81ac2fb..17131ee 100644
--- a/configure.ac
+++ b/configure.ac
@@ -71,7 +71,7 @@ AC_PROG_LN_S
 AM_DISABLE_STATIC
 AM_PROG_LIBTOOL
 IT_PROG_INTLTOOL([0.35.0])
-AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
+PKG_PROG_PKG_CONFIG
 
 GETTEXT_PACKAGE=gcr
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [The gettext domain name])
-- 
2.22.0
